src/Layout/Presentation/SingleDistrict/InfoBox.tsx

```diff
@@ -16,7 +19,7 @@ export class InfoBox extends React.Component<InfoBoxProps> {
                       " hadde nærmeste kvotient og trengte ",
                       vulnerable.moreVotesToWin,
                       " flere stemmer for å vinne mandatet. ",
-                          vulnerableVotes.partyCode,
+                          <b>{partyMap[vulnerableVotes.partyCode]}</b>,
```

Comment:
Codacy found an issue: [Missing "key" prop for element.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5947793)